CLI/3. Tutorials/2. 部署自動化 AIML pipeline.md

```diff
@@ -1 +1,215 @@
+
+### Step 1. Create SSH key
+- Enter the following command to create SSH key
+```
+$ ssh-keygen -t rsa -n 4096
+```
+
+Press <kbd>Enter</kbd> key to skip the following three questions
+
+```bash
+Enter file in which to save the key (/home/ubuntu/.ssh/id_rsa):
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+```
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_f96f4eb15d06d0905bc8b51f66f4878e.png)
+
+- Enter the following command to create a Interactive Container
+
+```bash
+$ twccli mk ccs -gpu 1
+```
+
+
+### Step 2. Insert SSH key to Interactive Container
+- Enter the following command to insert SSH key to Interactive Container
+
+```bash
+$ ssh-copy-id <USERNAME>@<CCS_IP> -p <PORT>
+
+#<USERNAME>:supercomputer account
+#<CCS_IP>:Public IP address of the Interactive Container
+#<PORT>:Container SSH's open port
+```
+
+- To insert the key in the container for the first time, you need to enter the **supercomputer password**. After the completion of the Virtual Computing Instance, the container will no longer need to enter the password:
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_4aba0afaefcbda5d00554ed19da64eac.png)
+
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note:** [<ins>Forgot supercomputer password?</ins>](https://man.twcc.ai/@twccdocs/guide-service-hostname-pwd-otp-zh#%E9%87%8D%E7%BD%AE%E4%B8%BB%E6%A9%9F%E5%AF%86%E7%A2%BC)
+:::
+
+
+### Step 3. Download the sample program:GPU Burn 
+
+This article uses the GPU Burn program as an operating example. You can refer to the following steps to load the required model training program after entering the container environment, or directly [Using SFTP + Filezilla ](https://www.twcc.ai/doc?page=hfs#%E4%BD%BF%E7%94%A8-SFTP--Filezilla-%E5%82%B3%E8%BC%B8%E6%AA%94%E6%A1%88) to upload the program to the container default storage space - Hyper File System.
+
+- Enter the following command to enter the container environment
+```bash
+$ ssh <USERNAME>@<CCS_IP> -p <PORT>
+
+#<USERNAME>:supercomputer account
+#<CCS_IP>:Public IP address of the Interactive Container
+#<PORT>:Container SSH's open port
+```
+
+- Download the sample program:GPU Burn
+```bash
+$ git clone https://github.com/wilicc/gpu-burn.git
+```
+
+- Compile the program into usable command (may be omitted)
+
+```bash
+$ cd gpu-burn
+$ make >> progress.log 2>&1
+```
+
+- Exit the container and return to the Virtual Computing Instance environment
+ 
+```bash
+$ exit
+```
+
+- After completing the above actions, you can delete the container used in the deployment environment. Please list all containers first and find out the container ID
+ 
+```bash
+$ twccli ls ccs
+```
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_fdc83baa8f0a580a72f1bb4935802ba2.png)
+
+
+
+- Delete the container of the deployment environment
```

Comment:
[SUG]

Delete the container of the deployment environment > Delete the container for deploying the environment